project/Build.scala

```diff
@@ -1159,123 +1110,103 @@ object Build {
+      fatalWarningsSettings,
+      name := "scalajs-env-phantomjs",
+      libraryDependencies ++=
+        PhantomJSEnvPlugin.phantomJSJettyModules.map(_ % "provided"),
+      libraryDependencies +=
+        "com.novocode" % "junit-interface" % "0.9" % "test"
   ).dependsOn(jsEnvs, jsEnvsTestKit % "test")
 
-  lazy val phantomJSEnvPlugin: Project = Project(
-      id = "phantomJSEnvPlugin",
-      base = file("phantomjs-sbt-plugin"),
-      settings = (
-          commonSettings ++ publishIvySettings ++ fatalWarningsSettings
-      ) ++ Seq(
-          name := "sbt-scalajs-env-phantomjs",
-          sbtPlugin := true,
-          scalaBinaryVersion :=
-            CrossVersion.binaryScalaVersion(scalaVersion.value),
-
-          // Add API mappings for sbt (seems they don't export their API URL)
-          apiMappings ++= {
-            val deps = (externalDependencyClasspath in Compile).value
-            val sbtJars = deps filter { attributed =>
-              val p = attributed.data.getPath
-              p.contains("/org.scala-sbt/") && p.endsWith(".jar")
-            }
-            val docUrl =
-              url(s"http://www.scala-sbt.org/${sbtVersion.value}/api/")
-            sbtJars.map(_.data -> docUrl).toMap
-          }
-      )
+  lazy val phantomJSEnvPlugin: Project = (project in file("phantomjs-sbt-plugin")
+  ).settings(
+      commonSettings,
+      publishIvySettings,
+      fatalWarningsSettings,
+      name := "sbt-scalajs-env-phantomjs",
+      sbtPlugin := true,
+      scalaBinaryVersion :=
+        CrossVersion.binaryScalaVersion(scalaVersion.value),
+
+      // Add API mappings for sbt (seems they don't export their API URL)
+      apiMappings ++= {
+        val deps = (externalDependencyClasspath in Compile).value
+        val sbtJars = deps filter { attributed =>
+          val p = attributed.data.getPath
+          p.contains("/org.scala-sbt/") && p.endsWith(".jar")
+        }
+        val docUrl =
+          url(s"http://www.scala-sbt.org/${sbtVersion.value}/api/")
+        sbtJars.map(_.data -> docUrl).toMap
+      }
   ).dependsOn(plugin, phantomJSEnv)
 
   // Examples
 
-  lazy val examples: Project = Project(
-      id = "examples",
-      base = file("examples"),
-      settings = commonSettings ++ Seq(
-          name := "Scala.js examples"
-      )
+  lazy val examples: Project = project.settings(
+      commonSettings,
+      name := "Scala.js examples"
   ).aggregate(helloworld, reversi, testingExample)
 
   lazy val exampleSettings = commonSettings ++ myScalaJSSettings ++ fatalWarningsSettings
 
-  lazy val helloworld: Project = Project(
-      id = "helloworld",
-      base = file("examples") / "helloworld",
-      settings = exampleSettings ++ Seq(
-          name := "Hello World - Scala.js example",
-          moduleName := "helloworld",
-          scalaJSUseMainModuleInitializer := true
-      )
+  lazy val helloworld: Project = (project in file("examples") / "helloworld"
+  ).settings(
```

Comment:
That one is particularly confusing. Could you add `()` around `file("examples") / "helloworld"`, because the precedence is not clear, here.